ui/store/actions.ts

```diff
@@ -4665,6 +4678,33 @@ export function hideNetworkBanner() {
   return submitRequestToBackground('setShowNetworkBanner', [false]);
 }
 
+export function setSwitchedNetworkDetails(switchedNetworkDetails) {
+  return submitRequestToBackground('setSwitchedNetworkDetails', [
+    switchedNetworkDetails,
+  ]);
+}
+
+export function neverShowSwitchedNetworkMessage() {
+  return submitRequestToBackground('setSwitchedNetworkNeverShowMessage', [
+    true,
+  ]);
+}
+
+// TODO: codeword NOT_A_THUNK @brad-decker
+export function setTransactionSecurityCheckEnabled(
```

Comment:
Was this included by mistake?

Reply:
Yep!  No idea how this snuck into a rebase!